src/reactviews/pages/QueryResult/queryResultStateProvider.tsx

```diff
@@ -41,7 +45,50 @@ const QueryResultStateProvider: React.FC<QueryResultContextProps> = ({
                             tabId: tabId,
                         });
                     },
+                    getExecutionPlan: function (
```

Comment:
If these are copied from the execution plan provider, it will be good to have one implementation of it.

Reply:
made an issue to track: https://github.com/microsoft/vscode-mssql/issues/18186